src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/TransferClassificationActivity.java

```diff
@@ -19,4 +22,16 @@ public interface TransferClassificationActivity {
 	 */
 	@ActivityMethod
 	void classify(Long orgId, String iuv, String iur, int transferIndex);
+
+	/**
+	 * retrieve Transfer data by semantic key
+	 *
+	 * @param orgId the unique identifier of the organization
+	 * @param iuv   the unique identifier of the payment (IUV)
+	 * @param iur   the identifier of the receipt associated with the payment
+	 * @param transferIndex  the index of the transfer to be classified
+	 * @return List of PaymentsReportingDTO object containing data found (may be 1 object)
+	 */
+	@ActivityMethod
+	List<PaymentsReportingDTO> retrievePaymentReportingBySemanticKey(Long orgId, String iuv, String iur, int transferIndex);
```

Comment:
you should not add a new method on the activity, you should just call the findBySemanticKey of PaymentsReportingDao inside the `classify` method

Reply:
OK